src/plugins/data/public/ui/filter_bar/filter_options.tsx

```diff
@@ -179,13 +179,13 @@ class FilterOptionsUI extends Component<Props, State> {
         panelPaddingSize="none"
         repositionOnScroll
       >
-        <EuiPopoverTitle>
+        <EuiPopoverTitle paddingSize="s">
```

Comment:
Do you think its better to do these locally rather than set a default through OUI?

Reply:
Due to the potential blast radius, I think we should localize these changes. I am also concerned about different fonts and font-sizes across themes and their impact on all of these.